beets/autotag/hooks.py

```diff
@@ -598,6 +598,16 @@ def tracks_for_id(track_id):
             yield t
 
 
+def invoke_mb(call_func, *args):
+    if not config["musicbrainz"]["enabled"]:
+        return ()
```

Comment:
I do sort of like @wisp3rwind's suggestion to consider moving this out of the `invoke_mb` utility—it might make it a little clearer what's going on!

Reply:
I think I'm also fine with the current version (renamed to `invoke_mb`); I was mostly concerned about `handle_exc` being actively misleading since it does more than handling exceptions. So, from my side, I'd say it's up to you @snejus which variant you prefer.